migrations/users/tag-challenge-field-string2bool.js

```diff
@@ -0,0 +1,77 @@
+import monk from 'monk'; // eslint-disable-line import/no-extraneous-dependencies
+
+const migrationName = 'tag-challenge-field-string2bool.js';
+
+const connectionString = 'mongodb://localhost:27017/habitica-dev?auto_reconnect=true'; // FOR TEST DATABASE
+
+const dbUsers = monk(connectionString).get('users', { castIds: false });
+
+const progressCount = 1000;
+let count = 0;
+
+export default async function processUsers () {
+  const query = {
+    migration: { $ne: migrationName },
+    tags: {
+      $elemMatch: {
+        challenge: {
+          $exists: true,
+          $type: 'string',
+        },
+      },
+    },
+  };
+
+  while (true) { // eslint-disable-line no-constant-condition
+    // eslint-disable-next-line no-await-in-loop
+    const users = await dbUsers.find(query, {
```

Comment:
could you import the User model 

```
import { model as User } from '../../website/server/models/user';
```

and use that instead of `monk`? We stopped using it some time ago, you can see an example here https://github.com/HabitRPG/habitica/blob/develop/migrations/users/full-stable.js#L68